### PR TITLE
misc: added prettier and eslint configs

### PR DESCRIPTION
**Pull request by BenjaminPabst**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+Dockerfile
+docker-compose.yaml
+.vscode/
+.gitlab/
+.gitlab-ci.yml
+*.log
+*.sh
+src/app/shared/generated/**
+src/app/api/*
+src/app/model/*
```
Comment (bastianjakobi) [inline .prettierignore:24]: What are those two going to contain?

File: src/index.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```
Comment (bastianjakobi) [inline src/index.html:1]: VSCode, by default, generates an uppercase DOCTYPE when creating an HTML file. Doctype is case-insensitive in HTML so it doesn't really matter, but was there any specific reasoning behind this change?

**Review by bastianjakobi — APPROVED** [review]
LGTM!

File: src/index.html (hunk 2 above)
Comment (BenjaminPabst) [reply src/index.html:1]: this is done by eslint/prettier

**Pull request merged**